drivers/pwm/pwm_led_esp32.c

```diff
@@ -1,13 +1,12 @@
 /*
  * Copyright (c) 2017 Vitor Massaru Iha <vitor@massaru.org>
- * Copyright (c) 2022 Espressif Systems (Shanghai) Co., Ltd.
+ * Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
```

Comment:
maybe update to 2025 as well as other updated files, such as binding. Otherwise LGTM